common/value.ts

```diff
@@ -16,6 +16,17 @@ export enum ValueKind {
   BIGINT = 7,
 }
 
+const VALUE_KIND_NAMES = [
+  'String',
+  'Int',
+  'BigDecimal',
+  'Bool',
+  'Array',
+  'Null',
+  'Bytes',
+  'BigInt',
+]
```

Comment:
Shouldn't `Bool` and `Null` be lowercase?

Reply:
Wasn't sure how that's shown elsewhere. Just changed it.